src/components/ButtonGroup/ButtonGroup.jsx

```diff
@@ -107,7 +108,8 @@ const ButtonGroup = React.createClass(createLucidComponentDefinition({
 							isActive={_.includes(selectedIndices, index)}
 							{...buttonChildProp}
 							key={index}
-							onClick={_.partial(this.handleSelect, index, buttonChildProp.onClick)}
+							uniqueId={index}
```

Comment:
- [ ] fix ButtonGroup to handle children uniqueIds better
- [ ] fix RadioGroup to handle children uniqueIds better